app/controllers/games_controller.rb

```diff
@@ -14,9 +16,17 @@ def show
     render text: 'Not Found', status: :not_found if @game.blank?
   end
 
+  def update
+    @game = Game.find(params[:id])
+    if @game.black_player != current_user
```

Comment:
Is this just a sanity check to potentially avoid an unnecessary database call? cool.